sherpa/astro/xspec/tests/test_xspec.py

```diff
@@ -37,7 +37,7 @@
 #    '(XSConvolutionKernel)'
 # in `xspec/__init__.py`
 #
-XSPEC_MODELS_COUNT = 281
+XSPEC_MODELS_COUNT = 289
```

Comment:
We went from 28 to 289 models?
```suggestion
XSPEC_MODELS_COUNT = 29
```

Reply:
Errr, no. We went from 281 to 289 models. As the code originally said.

Reply:
Oh wow. That's a lot of models!

Reply:
For reference, in CIAO 4.17 (which only supports models up to 12.14.0) we have

```
sherpa In [3]: len(list_models("xspec"))
Out[3]: 281

sherpa In [4]: list_models("xspec")[0:5]
Out[4]: ['xsabsori', 'xsacisabs', 'xsagauss', 'xsagnsed', 'xsagnslim']

sherpa In [5]: from sherpa.astro import xspec

sherpa In [6]: xspec.get_xsversion()
Out[6]: '12.14.0k'

```